product-service/handlers/product-new.js

```diff
@@ -0,0 +1,36 @@
+const {Client} = require('pg');
+
+const dbOptions  = {
+  host: 'lesson4.czoz6duffqoz.eu-central-1.rds.amazonaws.com',
+  port: 5432,
+  database: 'lesson4',
+  user: 'postgres',
+  password: '',
+  ssl: {
+    rejectUnauthorized: false
+  },
+  connectionTimeoutMillis: 5000
+};
+
```

Comment:
Consider declaring configs for db in the separate file and import it wherever you need. Consider using env variables instead of hardcoded data.